script.js

```diff
@@ -1,4 +1,553 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+/*PROJECT 3: BLACKJACK
+Source: https://fundamentals.rocketacademy.co/projects/project-3-blackjack */
+
+// Create a global array for a deck of 52 cards. All picture cards have rank of 10 (based on Blackjack rules).
+var deck = [
```

Comment:
Consider using the function that creates a deck. Hardcoding is never recommended.